src/cable/monkeypatch/redis.cr

```diff
@@ -0,0 +1,13 @@
+# On redis shard it tries to convert the return of command to Nil
+# When returning an array, it raises an exception
+# So we mokeypatch to run the command, ignore it, and retun Nil
+class Redis
+  module CommandExecution
+    module ValueOriented
+      def void_command(request : Request) : Nil
+        command(request)
+        Nil
```

Comment:
humm I was having problem with the [redis conversion](https://github.com/stefanwille/crystal-redis/blob/009e35fa40bbd518798afcc32c397402c6c6acb2/src/redis/command_execution/value_oriented.cr#L80-L82), but you are right, no `Nil` needed

Reply:
Oh, I just noticed the code. It's because they are casting to `nil` `.as(Nil)`. Removing that cast says "return as you return, but this method is nil".